### DISCUSSION TITLE
Fix Registration

### DISCUSSION THREAD

**Discussion by ChristophorosFeggaris** [Budibase/budibase#3353] [Help]
Hi everyone,

I'm new in Budibase platform. I just tried to register to the platform and after that I deleted the budibase account.Now the platform does not allow me to sign up. It directs me to verify page and i don't know what to do to make a new account with a new mail.

**Comment by mjashanks** [comment]
Hey @ChristophorosFeggaris - we'll look into this - thank you! 

In the meantime - have you tried clearing cookies or signing up in private browsing mode?

**Reply by shogunpurple** [reply]
If not using incognito - clear the `localStorage` in your browser. What has happened here is that the localStorage has not been cleared when you signed up, so when you deleted your account and went to re-sign up, it thinks you are at a certain stage in the process. Clearing your `localStorage` will fix it, and we will look into the root cause. Thanks!
